.github/pull_request_template.md

```diff
@@ -0,0 +1,38 @@
+# Description
+
+Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
```

Comment:
I guess all these can be made as comments instead of actual text so that people do not need to remove them to add their content

Reply:
But then no one can read those comments.

Reply:
The person editing the template would be able to.
That's how templates are made in general.

Reply:
@rajivharlalka Should I comment out the whole doc?🤔

Reply:
https://raw.githubusercontent.com/vitessio/vitess/main/.github/pull_request_template.md

Checkout this as an example.